### ISSUE TITLE
Transfer fuel between fuelcargo sources

### ISSUE THREAD

**Issue by partycrashr** [acemod/ACE3#8386]
It seems this has been a request in other forums (and has definitely been on my mind for a few years). Is is possible to add code/add an option to transfer fuel between fuelcargo sources? For example, transfer fuel from a fuel bladder to a fuel truck tank and vis versa? I was thinking either directly between the two objects (such as adding a fuelcargo refuel point), or maybe creating an additional placeable object that handles the transfer of fuel between the fuelcargo sources? Maybe the placeable object is a pump that is connected with hoses to both items?

The pumps could be linked to cover longer distances (for example, the 600 GPM pump can only pump so far by itself, but if you place additional pumps along the hose they can pump further). Opens up a lot of possibilities for logistics based gameplay, allowing for transferring fuel between locations as well as the setup of fuel farms. We could get into much greater detail using valves/meters, fuel filters/fuel age/need to recirculate the fuel, etc.

Basically, what my desire for this feature is to be able to accurately simulate transferring fuel from "ship" to shore and beyond as part of logistics for a MEU unit I want to setup. Different fuel types would be cool too for the different vehicles (jets/helicopters with JET A vs JP8 diesel for most ground vehicles, etc), but that might be too big an ask lol. However, at a minimum all I need to be able to do is transfer fuel between fuel storage tanks though.

**Comment by BrettMayson** [comment]
Why did you recreate an issue with the exact same text? As Jokoho said
> feature-request belongs into the feature-request issue #3594

**Comment by partycrashr** [comment]
So I create a comment on that thread?  Sorry, I was confused.

**Comment by BrettMayson** [comment]
Yes, features requests just get commented on that, rather than being their own issues.
